src/test/ui/bad/bad-mid-path-type-params.stderr

```diff
@@ -1,27 +1,48 @@
-error[E0107]: wrong number of type arguments: expected 1, found 2
-  --> $DIR/bad-mid-path-type-params.rs:30:28
+error[E0107]: wrong number of type arguments for `S::<T>::new`: expected 1, found 2
+  --> $DIR/bad-mid-path-type-params.rs:30:13
    |
+LL |     fn new<U>(x: T, _: U) -> S<T> {
+   |     ----------------------------- defined here, with 1 type parameter: `U`
+...
 LL |     let _ = S::new::<isize,f64>(1, 1.0);
-   |                            ^^^ unexpected type argument
+   |             ^^^^^^        ---- help: consider removing this type argument
```

Comment:
I wonder if we should use slightly more forceful wording: "remove this type parameter"

Reply:
Initially I was trying to remain consistent with `consider introducing a named lifetime parameter`, but yeah - `remove` feels better after all :-)

Reply:
changed 🎉